modern-cpp-font-lock.el

```diff
@@ -163,6 +163,65 @@ http://en.cppreference.com/w/cpp/language/integer_literal"
   :type 'boolean
   :group 'modern-c++-font-lock)
 
+;; Binary literal integers
+(defcustom modern-c++-literal-binary-integer-prefix-face 'font-lock-keyword-face
```

Comment:
I find the wording fine, maybe a bit too long. You can shorten it a bit by dropping `integer`, for example: `modern-c++-literal-binary-prefix-face` (and apply it to other new variables).

Why do you find the wording bad?

Reply:
`prefix`, `infix` and `suffix` are not semantic